README.md

```diff
@@ -11,6 +11,17 @@ The output is in `build/`.
 
 Note: if you change layouts, you'll need to run `make` again as the watcher fails to watch them :/
 
+## Linkchecker
+The "linkchecker" verifies that no dead links are present on the website. 
+`linkchecker` is a python app that can be acquired through pip by running 
+`pip install LinkChecker`. To verify that there aren't any dead links present
```

Comment:
oof. how complicated is it? could we have a go replacement?

also, ipfs is reliable enough to start "vendoring" deps to ipfs itself.

(having to manually run pip is frustrating and complicates usage and CI)